style.css

```diff
@@ -63,4 +63,367 @@
   .icons {
     padding-right: 1.5rem;
   }
+
+  .projects {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem;
+    gap: 0.75;
+    width: 20.438rem;
+    height: 34.5rem;
+    background: #fff;
+    border: 0.063rem solid #dfe1e6;
+    border-radius: 1rem;
```

Comment:
- [ ] I would kindly suggest you to Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels its best practices and will help your website responsive.